### ISSUE TITLE
Long text inside the table won't rendered

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jkoo7yeha/dompdf#297]
```
What steps will reproduce the problem?
1. create long text inside the table
2. tested in the dompdf demo
3.

What is the expected output? 
I want the long text inside the table

What do you see instead?
The render result got blank

What version of the product are you using? 
latest source in the svn trunk downloaded today

On what operating system?
Mac OSX

Please provide any additional information below.
When I render 4 paraghraps  inside the table it was ok, until I add new 
paragrap with no result.
So is there any limitation character length? Here I also attach the html test 
file.
```

Original issue reported on code.google.com by `heny...@gmail.com` on 1 Jun 2011 at 8:26
- Merged into: #91

**Comment by GoogleCodeExporter** [comment]
```
Sounds like your table cell is larger than the page size. This is a known 
problem, see issue 91. We hope to have this addressed for the 0.6.0 final 
release.
```

Original comment by `eclecticgeek` on 2 Jun 2011 at 2:54
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
Original comment by `eclecticgeek` on 30 May 2013 at 5:15
- Added labels: **Restrict-AddIssueComment-Commit**
